website/content/en/docs/deployment/cognito/guide-automated.md

```diff
@@ -119,7 +118,7 @@ This guide assumes you have Python 3.8 installed and that you have completed the
     kustomize build deployments/cognito | kubectl delete -f -
     ```
 1. To delete the rest of resources(subdomain, certificates etc.), run the following commands from the root of your repository:
-> Note: Make sure that you have the configuration file created by the script in `tests/e2e/utils/cognito_bootstrap/config.yaml`. If you did not use the script, plug in the name, ARN, or ID of the resources that you created in a yaml file in `tests/e2e/utils/cognito_bootstrap/config.yaml` by referring to the following sample:
+    > Note: Make sure that you have the configuration file created by the script in `tests/e2e/utils/cognito_bootstrap/config.yaml`. If you did not use the script, plug in the name, ARN, or ID of the resources that you created in a yaml file in `tests/e2e/utils/cognito_bootstrap/config.yaml` by referring to the following sample:
```

Comment:
thx, can you also check the note in line 151?

Reply:
looks fine : https://akartsky.github.io/kubeflow-manifests/main/docs/deployment/cognito/guide-automated/